src/model/node.ts

```diff
@@ -0,0 +1,5 @@
+import { Point } from "./point"
+
+export interface Node {
+  point: Point
```

Comment:
Did we also want to have an id here?

```suggestion
  point: Point
  id: string
```

Reply:
(I'm now not sure it's necessary — I'm just using the Node's identity, rather than a string ID. Seems fine for now.)